src/blindpsbt.cpp

```diff
@@ -0,0 +1,430 @@
+// Copyright (c) 2017-2019 The Elements Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <blindpsbt.h>
+
+#include <hash.h>
+#include <primitives/transaction.h>
+#include <primitives/confidential.h>
+#include <psbt.h>
+#include <issuance.h>
+#include <random.h>
+#include <util/system.h>
+
+static secp256k1_context* secp256k1_blind_context = NULL;
```

Comment:
Is there anyway we can reuse the static context from blind.cpp? Each of these costs like 5Mb, which starts to add up in CI when running 4 multi-daemon functional tests in parallel. (Though I guess we have only one or two tests which use PSET so maybe it'll be ok..)

Reply:
Done (I think).